sdk/analytics/src/main/java/com/klaviyo/analytics/model/Event.kt

```diff
@@ -23,7 +23,7 @@ class Event(val metric: EventMetric, properties: Map<EventKey, Serializable>?) :
         get() = when (val value = this[EventKey.VALUE]) {
             is Double -> value
             else -> try {
-                value.toString().toDouble()
+                value?.toString()?.toDouble()
```

Comment:
this was causing a lower priority issue, which is just that any event that had no `value` set would hit the exception below, because instead of returning `null` here, we'd cast to `"null"` then hit a number format exception 🤦